### ISSUE TITLE
Skip Olduvis cause ain't nobody got time fo' dat

### ISSUE THREAD

**Issue by Y0lk** [nospoone/holobunnies-issues#24]
You can go to the right, jet pack straight up to the exit, and it crashes the game:

```haxe
openfl._legacy.display.Stage.__pollTimers (openfl/_legacy/display/Stage.hx line 1084)
openfl._legacy.display.Stage.__checkRender (openfl/_legacy/display/Stage.hx line 351)
openfl._legacy.display.Stage.__render (openfl/_legacy/display/Stage.hx line 1103)
openfl._legacy.display.DisplayObjectContainer.__broadcast (openfl/_legacy/display/DisplayObjectContainer.hx line 286)
openfl._legacy.display.DisplayObject.__broadcast (openfl/_legacy/display/DisplayObject.hx line 161)
openfl._legacy.display.DisplayObject.__dispatchEvent (openfl/_legacy/display/DisplayObject.hx line 182)
openfl._legacy.events.EventDispatcher.dispatchEvent (openfl/_legacy/events/EventDispatcher.hx line 98)
flixel.FlxGame.onEnterFrame (flixel/FlxGame.hx line 541)
flixel.FlxGame.step (flixel/FlxGame.hx line 674)
flixel.FlxGame.update (flixel/FlxGame.hx line 750)
flixel.FlxState.tryUpdate (flixel/FlxState.hx line 168)
states.BossRushPlay.update (states/BossRushPlay.hx line 211)
flixel.group.FlxTypedGroup.forEach (flixel/group/FlxGroup.hx line 691)
*._Function_1_3 (states/BossRushPlay.hx line 318)
```

**Comment by nospoone** [comment]
Duplicate of #17.

**Comment by nospoone** [comment]
#octavius

**Comment by Y0lk** [comment]
Olduvis*, oops

**Comment by nospoone** [comment]
LOL - seemed weird that you were reopening the same issue. I get it now. Related to #17, but not the same thing :+1:

**Comment by nospoone** [comment]
This should be fixed in `0.0.3`. Please reopen if you encounter this again!
